.github/workflows/tests.yml

```diff
@@ -42,7 +42,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest] # [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.7, 3.8, 3.9, "3.10"]
+        python-version: ["3.7.3", 3.7, 3.8, 3.9, "3.10"]
```

Comment:
just curious about the reason for explicitly including 3.7.3?

Reply:
It's because some of the imports are for cases < 3.7.4. Those stopped being covered when we removed 3.6 in #56.

https://app.codecov.io/gh/drivendataorg/erdantic/compare/56